routerchain/assetlist.json

```diff
@@ -0,0 +1,39 @@
+{
+    "$schema": "../assetlist.schema.json",
+    "chain_name": "routerchain",
+    "assets": [
+      {
+        "description": "The native token of Router Chain",
+        "extended_description": "Router Chain is a PoS L1 that allows stateful bridging–enabling truly omnichain dapps that can abstract chains from the users and aggregate liquidity across multiple chains.",
+        "denom_units": [
+          {
+            "denom": "route",
+            "exponent": 0
+          },
+          {
+            "denom": "ROUTE",
+            "exponent": 18
+          }
+        ],
+        "base": "route",
+        "name": "Router Chain",
+        "display": "route",
```

Comment:
this display denom should now correspond to the unit to be displayed, which I presume is "ROUTE"